examples/.gitignore

```diff
@@ -0,0 +1,3 @@
+node/package-lock.json

```

Comment:
Purely making a note here so I remember to remove the `examples` directory - would be great if we could turn these into tutorials :)

Reply:
Yeah honestly the examples don't make a ton of sense as part of the lib core, but that's where I run the tests, so I don't know if it's feasible to remove the examples completely 😬

Reply:
It was more to help both us understand if it works for each environment :) After all we could turn the 'setups' into written tutorials

though i see what you mean...might be worth something to you, just maybe there's a better place to put these directories?

Reply:
I might've actually misunderstood what you meant, my bad. Ignore my previous comment.

Reply:
resolving, pr desc now includes a note about this